hedera-node/hapi-utils/src/test/java/com/hedera/node/app/hapi/utils/ethereum/EthTxDataTest.java

```diff
@@ -601,4 +604,18 @@ void bigPositiveValueWithDifferentTypes(EthTransactionType type) {
 
         assertEquals(bigValue, populateEthTxData.value());
     }
+
+    @Test
+    void thatPassesOnBigIntegerByteArray() {
```

Comment:
nit: function name should be more descriptive to test case
```suggestion
    void populateEthTxDataWithLargeSignedChanId() {
```

Reply:
Finding a better name for this test is a great idea! However, I don’t want to focus solely on the id size, as that's not the only issue. The problem could occur even with smaller numbers. A more detailed explanation can be found here:

https://github.com/hashgraph/hedera-services/issues/15998

What do you think about names:
```
populateEthTxDataComparedToSignedByteArrayExtraByteAdded
populateEthTxDataComparedToUnsignedByteArrayNoExtraByteAdded
```
for these tests?

@david-bakin-sl 
@Nana-EC